examples/cmd/nse/dirty-nse/nse.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"time"
+
+	"github.com/networkservicemesh/networkservicemesh/sdk/endpoint"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
```

Comment:
> I am not sure why is this needed?
> Also, please add signal handling.

In this context **dirty** means that NSE doesn't perform any cleanup routines. There are a lot of cases when the process can suddenly stop and never continue the execution, so it is to simulate such cases.
So I believe there is no need to add some signal handling in here. The process is broken and so it should be killed by the signal received.

Reply:
Outdated